tests/test_datasets.py

```diff
@@ -64,6 +64,30 @@ def test_preprocessing_multiprocess(data_dir):
     dataset.load_custom_dataset_from_folder(data_dir + "/sample_texts")
 
 
+def test_preprocessing_nothing(data_dir):
```

Comment:
might rename, sounds like it's checking empty dataset instead (I like test_preprocessing_minimal)
might include a comment at the top or a docstring that indicates the purpose of the test